worker/migrations/migrations.go

```diff
@@ -80,6 +91,137 @@ func commit(ctx *job.WorkerContext, err error) error {
 	return err
 }
 
+// Migrate all the encrypted accounts to Bitwarden ciphers.
+// It decrypts each account, reencrypt the fields with the organization key,
+// and save it in the ciphers database.
+func migrateAccountsToOrganization(domain string) error {
+	inst, err := instance.GetFromCouch(domain)
+	if err != nil {
+		return err
+	}
+	setting, err := settings.Get(inst)
+	if err != nil {
+		return err
+	}
+	// Get org key
+	var orgKey []byte
+	orgKey, err = setting.OrganizationKey()
```

Comment:
[nit] We can avoid the orgKey declaration with `orgkey, err := setting.OrganizationKey()`